go/common/version/version.go

```diff
@@ -93,12 +91,6 @@ var (
 	// https://github.com/tendermint/tendermint/blob/1635d1339c73ae6a82e062cd2dc7191b029efa14/state/validation.go#L21-L22.
 	TendermintAppVersion = ConsensusProtocol.MaskNonMajor().ToU64()
 
-	// Tendermint exposes the tendermint core version.
-	Tendermint = parseSemVerStr(version.TMCoreSemVer)
```

Comment:
This makes https://github.com/oasisprotocol/oasis-core/issues/3402 obsolete, feel free to close that issue

Reply:
Yeah the reason is that the Tendermint Core version is no longer included as it is added at build time.